dolly-backend-app/src/main/java/no/nav/dolly/service/TestgruppeService.java

```diff
@@ -52,6 +56,15 @@ public Testgruppe opprettTestgruppe(RsOpprettEndreTestgruppe rsTestgruppe) {
         );
     }
 
+    public RsTestgruppeMedBestillingId fetchPaginertTestgruppeById(Long gruppeId, Integer pageNo, Integer pageSize) {
+        Testgruppe testgruppe = fetchTestgruppeById(gruppeId);
+        Page<Testident> testidentPage = identService.getBestillingerFromGruppePaginert(gruppeId, pageNo, pageSize);
```

Comment:
Hvis dette kan løses med et databaseoppslag og ikke to så vil vi trolig få bedre ytelse/responstid. Ikke sikkert at det er vesentlig dog.

Reply:
Ja, tenkte på det, men ble så veldig komplekst siden den skulle hente hele identer kolonnen fra ident repository 😅